internal/service/kms/secrets_data_source_test.go

```diff
@@ -62,6 +86,27 @@ func testAccSecretsEncryptDataSource(key *kms.KeyMetadata, plaintext string, enc
 	}
 }
 
+func testAccSecretsEncryptDataSourceAsym(key *kms.KeyMetadata, plaintext string, encryptedPayload *string) resource.TestCheckFunc {
+	return func(s *terraform.State) error {
+		conn := acctest.Provider.Meta().(*conns.AWSClient).KMSConn
```

Comment:
I think adding the missing `()` should address the linting issues

```suggestion
		conn := acctest.Provider.Meta().(*conns.AWSClient).KMSConn()

Reply:
Done as Requested.